app/src/main/java/org/p2p/wallet/striga/offramp/models/StrigaOffRampTokenState.kt

```diff
@@ -0,0 +1,25 @@
+package org.p2p.wallet.striga.offramp.models
+
+import java.math.BigDecimal
+
+sealed class StrigaOffRampTokenState(
```

Comment:
this state is essentially extending the SwapWidgetModel to add more states in addition to `Content` and `Loading` and might need to be revised